src/main/java/uk/gov/hmcts/reform/prl/controllers/CallbackController.java

```diff
@@ -320,14 +324,18 @@ public AboutToStartOrSubmitCallbackResponse generateDocumentSubmitApplication(
                     "paymentServiceRequestReferenceNumber",
                     paymentServiceResponse.getServiceRequestReference()
                 );
+            } else {
+                caseDataUpdated.put("caseStatus", CaseStatus.builder()
+                    .state(SUBMITTED_PAID.getLabel())
+                    .build());
+                caseDataUpdated.put(STATE_FIELD, SUBMITTED_PAID.getValue());
```

Comment:
This else is the block for courtadmin (327-331) . why not put this as case created by courtadmin? and leave the existing else block as is?

Reply:
added this code in existing else condition retesting logic